metrics/naming.go

```diff
@@ -0,0 +1,16 @@
+package metrics
+
+const serviceName = "pulse"
+const requestsMetricBaseName = "requests"
+
+func metricName(name string) string {
+	return serviceName + "_" + name
```

Comment:
Maybe it's better to use some more generic name for the http metrics and just add label `app` to differ between applications.

Reply:
I hesitated between these two solutions... But I don't see any reason why I shouldn't not do it your way, if you find it beneficial. 
Check my new changes. Below it the example of how one metric-line would look like:
```
bux_requests_total{app="pulse",classification="success",method="GET",path="/some_path",status="200"} 4
```